torch/_dynamo/symbolic_convert.py

```diff
@@ -1747,36 +1829,64 @@ def create_call_resume_at(self, inst):
             for k in self.symbolic_locals.keys()
             if k in reads and k not in self.cell_and_freevars()
         )
-        nargs = len(self.stack) + len(argnames)
+
+        cg = PyCodegen(self)
+
+        # remove nulls from the stack and restore them in the
```

Comment:
Why do you have to remove them?

Reply:
Python doesn't allow NULL to be an argument to a function (and we eventually pass the entire stack to the resume function).